### PR TITLE
Tmp

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,18 +21,18 @@
   },
   "dependencies": {
     "chalk": "^1.0.0",
-    "lodash": "^3.2.0",
-    "maxmin": "^1.0.0",
-    "uglify-js": "2.4.17",
+    "lodash": "^3.6.0",
+    "maxmin": "^1.1.0",
+    "uglify-js": "^2.4.19",
```
Comment (vladikoff) [inline package.json]: there are issues with new uglify i assume?

Comment (vladikoff) [inline package.json]: do we need to wait until they fix it?

File: Gruntfile.js [hunk 2]
```diff
@@ -311,34 +311,7 @@ module.exports = function(grunt) {
...
-    'uglify:sourcemap_customRoot',
-    'uglify:sourcemap_functionName',
-    'uglify:sourcemap_multiple',
-    'uglify:sourcemap_multipleFunctionNames',
-    'uglify:sourcemapin',
-    'uglify:sourcemap_sources',
-    'uglify:sourcemapin_sources',
-    'uglify:expression_json',
-    'uglify:expression_js',
+    'uglify',
```
Comment (vladikoff) [inline Gruntfile.js]: this is fine

**Comment by XhmikosR** [comment]
@vladikoff: for some reason I didn't get notified of your comments at all :/

**Pull request closed without merging**